HogeFuga/library/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">Library</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/7bc72839ceeb91362b29abe3d6e45fdb9be34890/HogeFuga/library/src/main/res/values/strings.xml#L2"></span>The resource <code>R.string.app_name</code> appears to be unused</td>
    </tr>
  </tbody>
</table>